usaspending_api/download/helpers/elasticsearch_download_functions.py

```diff
@@ -85,9 +85,9 @@ def _populate_download_lookups(cls, filters: dict, download_job: DownloadJob, si
             download_job=download_job,
         )
 
-        DownloadJobLookup.objects.bulk_create(download_lookup_obj_list, batch_size=10000)
+        created_obj_list = DownloadJobLookup.objects.bulk_create(download_lookup_obj_list, batch_size=10000)
```

Comment:
What are we currently logging? 1?

Reply:
Currently logging the number of objects that we are trying to insert via `download_lookup_obj_list` but this change is to log the number of created objects that are returned from the `bulk_create` call.